nextcord/cache.py

```diff
@@ -0,0 +1,102 @@
+from time import time as now
+from typing import Optional, Union
+
+__set_item__ = dict.__setitem__
+__get_item__ = dict.__getitem__
+__get__ = dict.get
+__pop__ = dict.pop

```

Comment:
Yes, although this avoids dot narration which I tested was faster

Reply:
Does it really have that big of a speed difference? Using the base classes' methods in this manner makes the code less readable.

Reply:
Resolving as this results in a 2x+ significance in speedup. Micro-optimization of cache is beneficial as this will be reused in future PRs on cache reworks, these can be called millions of times per second, twice the speed is a major bump well worth slightly less readability in this case.
A detailed comment has been added to describe the reasoning.